common/source/docs/common-telemetry-rockblock.rst

```diff
@@ -28,36 +28,47 @@ of manual control (outside of low-rate GUIDED mode "fly to" locations) is not po
 The RockBLOCK has a latency of 20-60 seconds and is only able to send 1 message per 30 second period (or greater). This period is
 configurable via the ``RCK_PERIOD`` parameter.
 
+If desired, the RockBlock can automatically activate on loss-of-GCS-telemetry. This is done by setting the ``RCK_FORCEHL`` parameter to 2.
+The RockBLOCK will then automatically activate when telemetry is lost for ``RCK_TIMEOUT`` seconds. When telemetry is restored,
+the RockBLOCK will deactivate.
+
 .. note:: Any modem using the Iridium 9602 or 9603 chipset *should* work, though only the RockBLOCK 9603N modem has been tested.
 
-Supported MAVLink command messages
-==================================
+Supported MAVLink messages
+==========================
 
-- CMD_NAV_RETURN_TO_LAUNCH
-- CMD_NAV_LAND
-- CMD_NAV_TAKEOFF
-- CMD_NAV_VTOL_TAKEOFF
-- CMD_NAV_VTOL_LAND
-- CMD_DO_SET_MODE
-- CMD_MISSION_START
-- CMD_COMPONENT_ARM_DISARM
-- CMD_CONTROL_HIGH_LATENCY
+The following MAVLink messages are supported from the GCS to the vehicle:
+- COMMAND_LONG
+- COMMAND_INT
+- MISSION_ITEM_INT
+- MISSION_SET_CURRENT
 
-Setup
-=====
+Withing the COMMAND_LONG and COMMAND_INT messages, the following commands are supported:
```

Comment:
needs blank line after the group above to display both bulletlists correctly

Reply:
That _should_ now be fixed.